Adapter/SsiCache.php

```diff
@@ -48,31 +49,31 @@ public function __construct($token, RouterInterface $router, ControllerResolverI
     /**
      * {@inheritdoc}
      */
-    public function flushAll()
+    public function flushAll(): bool
```

Comment:
I hope these signature changes can be considered an acceptable BC-break

Reply:
You hope?

Reply:
Just found this: https://github.com/sonata-project/SonataPageBundle/blob/3.x/Cache/BlockSsiCache.php